### ISSUE TITLE
[a11y audit] Finding: Chatbot MHV Prescription Skill: 4.3.4 Support screen readers

### ISSUE THREAD

**Issue by SarahKay8** [department-of-veterans-affairs/va.gov-team#67309]
### Testing Areas

Operating System: **_Mac + Windows_**
Assistive Technology being used: **_Jaws + Voiceover_** 
Internet Browser: _**Chrome, Firefox, Safari**_ 
Device Type: **_Desktop_** 
[Staging Link](
https://dev.va.gov/contact-us/virtual-agent/
)
### Test User 
vets.gov.user+228@gmail.com

### Steps to recreate

Manually type response and send message in text input field with VO turned on

### Issue
Only last message is being read out loud with a screen reader. This behavior happens when a response is manually typed and sent in text input field. This behavior happens sometimes.

### Issue Description
When a chatbot sends a sequence of messages, and only the last message in the sequence is read out loud by a screen reader, it can be problematic for users who rely on screen readers to access content. 
**Main Caveat** - This is an automated chatbot meaning there would not be time for each message to announce if every message was set to announce immediately.

https://github.com/department-of-veterans-affairs/va.gov-team/assets/102324990/463989ec-5a3b-42ba-891c-955cd25744cc

Move focus to the first new message that appears (this will cause it to announce and place the user in a natural place to continue reading the rest of the new messages)
The user can navigate forwards to the other messages at their own pace
Once all the new messages have been read they will be back on the text input to send a reply
My understanding is that it's not possible to queue up the messages with any dynamic announcement (e.g. if you were not moving focus) because there's no way to know how long each individual user's device will take to announce each message.

### Other consideration

Because chatbot can sometimes send a large number of messages, consider implementing message pagination or a “load more” button. This allows users to manage and navigate through lengthy chat histories more easily.
Add `"load more"`button message pagination

**Comment by Joy-Elizabeth** [comment]
[https://app.zenhub.com/workspaces/va-virtual-agent-6011826e423fcf000eb231cb/issues/gh/department-of-veterans-affairs/va-virtual-agent/1188](url)

**Comment by Joy-Elizabeth** [comment]
Chatbot team is dependent on MSFT for this, and will be working to address this with them as a priority post Rx skill launch.

**Comment by eli-oat** [comment]
Note, the following shouldn't be taken to be an accurate representation of how this manifests on mobile. 

That said, using the iOS Simulator and the in-built accessibility inspector I can say that iOS (which isn’t to say VoiceOver, necessarily) has access to the full internal DOM of the chatbot history even when it is scrolled out of view.

This doesn’t mean that VoiceOver necessarily reads it all out, but it means using the accessibility inspector I’m able to access visually hidden information in the chat bot’s history…the gigantic caveat here is that the way that the accessibility inspector does this is to force a scroll, bringing the hidden info into focus.

In theory, this should be like hitting the tab key. What I’m not sure of, though, is if VoiceOver’s response would suggest there is any reason to do that.

If supported, more investigation on mobile is necessary to fully understand this behavior.

**Comment by anitadewitt-va** [comment]
<details><summary>Recordings of VoiceOver test shows the skipping of chat bubbles inconsistently.
</summary>

https://github.com/department-of-veterans-affairs/va.gov-team/assets/8662795/20e7295c-a92d-4ae3-8698-71fa9b189405

https://github.com/department-of-veterans-affairs/va.gov-team/assets/8662795/bfe25921-2f10-45c0-b2a9-df0463655f01

</details>

<details><summary>Recording of VoiceOver test shows behavior after adding delays between messages.
</summary>

https://github.com/department-of-veterans-affairs/va.gov-team/assets/8662795/e6714ebe-abfb-4802-a14e-2557ad96e780

</details>

**Comment by sara-amanda** [comment]
@eli-oat or @SarahKay8 - can this ticket be closed, based on your work with the chatbot team?

**Comment by SarahKay8** [comment]
@sara-amanda hmmm I think so?

**Comment by SarahKay8** [comment]
@anitadewitt-va cool to close?

**Comment by sara-amanda** [comment]
Adding @rileyorr for awareness, but it looks like we are still waiting for the ok to close this one? 
Moving to `Validate` until we hear back @NaomiPMC.
